### PR TITLE
add LICENSE

### PR DESCRIPTION
**Pull request by JonathanLorimer**

### CODE REVIEW

**Comment by soareschen** [comment]
Is there any example of other Nix projects that use GPLv3? Would it be better to use a license that is more common with other Nix projects, like BSD, Apache, or MIT? Here are some examples I found:

- https://github.com/NixOS/nixpkgs/blob/master/COPYING
- https://github.com/tweag/gomod2nix/blob/master/LICENSE
- https://github.com/input-output-hk/haskell.nix/blob/master/COPYING
- https://github.com/tweag/nickel/blob/master/LICENSE

**Comment by JonathanLorimer** [comment]
@soareschen went with MIT

File: README.md [hunk 1]
```diff
@@ -10,6 +10,10 @@
 <a href="https://nixos.org/">
   <img src="https://img.shields.io/static/v1?logo=nixos&logoColor=white&label=&message=Built%20with%20Nix&color=41439a&style=for-the-badge" alt="Built with nix" />
 </a>
+<a href="https://github.com/informalsystems/cosmos.nix/blob/master/LICENSE">
+  <img src="https://img.shields.io/badge/license-GPL%20v3.0-brightgreen.svg?style=for-the-badge" alt="License" />
```
Comment (soareschen) [inline README.md:14]: The license here has not been updated.

**Review by soareschen — APPROVED** [review]
Awesome!

**Pull request merged**